pkg/elb/cleanup.go

```diff
@@ -0,0 +1,121 @@
+package elb
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/aws/aws-sdk-go/service/elb/elbiface"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"github.com/aws/aws-sdk-go/service/elbv2/elbv2iface"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/cloudprovider"
+	awsprovider "k8s.io/kubernetes/pkg/cloudprovider/providers/aws"
+
+	"github.com/weaveworks/eksctl/pkg/eks"
+)
+
+const (
+	elbClassic = iota
+	elbV2      = iota
+)
+
+// CleanupLoadBalancers finds and deletes any dangling ELBs attached to a service
```

Comment:
nit: We could call it `Cleanup`, as the package is dedicated to ELBs, so it would be `elb.Cleanup` and `elb.CleanupLoadBalancers` read a little redundant....